README.md

```diff
@@ -46,6 +46,22 @@ The project is using the Gradle Versions plugin to do a lot of heavy lifting her
 
 *Note that you'll still need to manually check for dependency updates on all buildscript plugins (see `Dependencies.kt` -> `Config.BuildScriptPlugins` properties), including the Gradle Versions plugin.*
 
+##Adding Tabs to the Nav Drawer
```

Comment:
@LuisRocketRangel nit: Add a space between the heading indicators and the heading text
```suggestion
## Adding Tabs to the Nav Drawer
```